doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -786,3 +787,351 @@ JCL      - fd5cc89 based on jdk-11.0.2+7)
   - *This product includes cryptographic software written by Eric Young (eay@cryptsoft.com).*
 
 :penguin: *Congratulations!* :tada:
+
+----------------------------------
+
+## Riscv64
+:rocket:
+The following instructions guide you through the process of building an **OpenJDK V11** binary that contains Eclipse OpenJ9 for Riscv64 (RISC-V 64-bit) Linux.
+
+:bulb:
+It is assumed that Ubuntu 16.04 is the default host system for the cross-compilation.
+
+### 1. Prepare your system for cross-compilation
+:rocket:
+A number of software packages/dependencies must be installed on the host system to create a suitable cross-compiling environment:
+
+- [The GNU cross-toolchain](https://github.com/riscv/riscv-gnu-toolchain), which contains the source code of the C/C++ cross-compiler intended for RISC-V.
+- [bootJDK](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=hotspot), which is only used in the cross-compilation on the host system.
```

Comment:
Why does this recommend the `hotspot` variant?

Reply:
Fixed the wrong link and replaced with an OpenJ9-based bootJDK as it will be used to populate the legacy java8 package data at `/openj9-openjdk-jdk11/src/java.base/share/classes/jdk/internal/module/jdk8_packages.dat` to the cross-build in the final stage of the cross-compilation (the java8 package information is different between OpenJ9 and Hotspot)

Reply:
The link should be to the latest release for a more reliable build: https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=openj9

Reply:
Fixed as suggested.